common/source/docs/common-autopilots.rst

```diff
@@ -88,4 +92,101 @@ available if you're still working on those platforms:
    PX4FMU
    Qualcomm Snapdragon Flight Kit
 
+.. _common-autopilots_limited_firmware:
+
+Firmware Limitations on Selected Hardware
+-----------------------------------------
+
+The ArduPilot firmware in some configurations exceeds 1 MB in size. Some autopilots may not have enough
+flash memory to store the full firmware.
+
+For the affected autopilots, a minimal firmware is generated. This firmware omits less-commonly used features
+in order to reduce the firmware size to less than 1 MB.
+
+Miminal firmware is required for the KakuteF7, KakuteF7 Mini, OmnibusF7V2, sparky2 and older versions of
+the Pixhawk (with the RevA, RevY and Rev1 of the STM32F427 chip).
+
+The missing features are listed below.
+
+
+-  **Common to all vehicles**
+
+   -  Aux function for testing IMU failover (KILL_IMU)
+   -  LTM, Devo and Hott telemetry formats
+   -  Piccolo CAN
+   -  Oreo LED lights
+   -  NCP5623 LED lights
+   -  NMEA output format
+   -  Solo Gimbal
+   -  DSP support
+   -  MTK, SIRF GPS support
+   -  EFI engine support
+   -  AK09916 on ICM20948 compass
+   -  Runcam
+   -  External I2C barometers
+   -  DLVR Airspeed sensors
+
+
+-  **Copter only**
+
+   -  Sprayer
+   -  Visual Odometry
+   -  Gripper
+   -  RPM
+   -  ADSB
+   -  Guided, Follow, Sport, SystemID, Zigzag and Autorotate modes
+   -  Beacon
+   -  OAPathPlanner
+   -  Optical Flow
+
+
+-  **Plane Only**
+
+   -  HIL
+   -  Gripper
+   -  Soaring
+   -  Landing Gear
+   -  Qautotune mode
+
+
+-  **Rover Only**
+
+   -  N/A
+
+
+-  **Sub Only**
+
+   -  N/A
+
+
+The following autopilots have specific minimal feature sets with the listed features not present.
```

Comment:
perhaps "have different reduced feature sets"

@tridge (again raising the question, why others have so many more removed?)

Reply:
Fixed